mmv1/products/edgenetwork/InterconnectAttachment.yaml

```diff
@@ -0,0 +1,134 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'InterconnectAttachment'
+description: |
+  A Distributed Cloud Edge interconnect attachment, which connects routers to the northbound network.
+references:
+  guides:
+    'Create and manage interconnect attachments': 'https://cloud.google.com/distributed-cloud/edge/latest/docs/attachments#api'
+  api: 'https://cloud.google.com/distributed-cloud/edge/latest/docs/reference/network/rest/v1/projects.locations.zones.interconnectAttachments'
+docs:
+base_url: 'projects/{{project}}/locations/{{location}}/zones/{{zone}}/interconnectAttachments'
+self_link: 'projects/{{project}}/locations/{{location}}/zones/{{zone}}/interconnectAttachments/{{interconnect_attachment_id}}'
+create_url: 'projects/{{project}}/locations/{{location}}/zones/{{zone}}/interconnectAttachments?interconnectAttachmentId={{interconnect_attachment_id}}'
+immutable: true
+import_format:
+  - 'projects/{{project}}/locations/{{location}}/zones/{{zone}}/interconnectAttachment/{{interconnect_attachment_id}}'
+  - '{{name}}'
+timeouts:
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 30
+autogen_async: true
+async:
+  actions: ['create', 'delete', 'update']
+  type: 'OpAsync'
+  operation:
+    base_url: '{{op_id}}'
+    path: 'name'
+    wait_ms: 1000
+  result:
+    path: 'response'
+    resource_inside_response: false
+  error:
+    path: 'error'
+    message: 'message'
+examples:
+  - name: 'edgenetwork_interconnect_attachment'
+    primary_resource_id: 'example_interconnect_attachment'
+    vars:
+      edgenetwork_interconnect_attachment_id: 'example-interconnect-attachment'
+      edgenetwork_network_id: 'example-network'
+    test_env_vars:
+      location: 'REGION'
+      zone: 'ZONE'
+    exclude_test: true
```

Comment:
Is there a specific reason we need to skip the test?

Reply:
If there are no other issues, will you please approve this PR so I can merge it?